codegen/src/main/java/software/amazon/awssdk/codegen/AddOperations.java

```diff
@@ -73,7 +74,7 @@ private static boolean isBlobShape(Shape shape) {
      * @return True if shape is a String type. False otherwise
      */
     private static boolean isStringShape(Shape shape) {
-        return shape != null && "String".equals(shape.getType());
+        return shape != null && shape.getType().equalsIgnoreCase("String");
```

Comment:
Let's keep the original ordering and compare to the constant: `"String".equalsIgnoreCase(shape.getType())`

Reply:
+1